utils/constants.go

```diff
@@ -71,6 +70,12 @@ var (
 		HashFallback:     kong.String("none"),
 		HashOnCookiePath: kong.String("/"),
 	}
+	defaultsRestrictedFields = map[string][]string{
+		"service": {"ID", "Host", "Name", "Port"},
+		"route": {"ID", "Name"},
+		"target": {"ID"}, // TODO check what is target4..
```

Comment:
`target4` may easily be a typo, I think harry was referring to the `target` field of the `target` entity.